pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -280,6 +280,11 @@ type ScrapeConfigSpec struct {
 	// If unset, Prometheus uses true by default.
 	// +optional
 	EnableCompression *bool `json:"enableCompression,omitempty"`
+	// When false, Prometheus will request uncompressed response from the scraped target.
+	//
+	// If unset, Prometheus uses true by default.
```

Comment:
nit: remove this line

Reply:
I updated the description to match another EnableHTTP2 instead.

Reply:
aah sorry, we do have a few places which explicitly states Prometheus uses certain default value.
May be we should remove that, since this is rare scenario like bool value is true by default its ok to keep.
But I don't have a strong opinion.

@simonpasquier wdyt?